kolibri/core/assets/src/views/custom-ui-menu/menu-option.vue

```diff
@@ -77,6 +78,10 @@
         type: Boolean,
         default: false,
       },
+      iconComponent: {
+        type: Object,
+        required: false,
+      },
```

Comment:
this seems a bit hacky...

What do you think of deprecating the `custom-ui-menu` altogether? We can fork it into two single-purpose versions: one for the side-nav, and one for the user-menu dropdown...

Reply:
Another option would be to define nav components instead of icon components, but an optional icon `slot` in the menu-option and have the nav components insert the SVG in there.

Reply:
(so the nav components become a thin wrapper around menu-option.vue)

Reply:
migrating this to a new issue here #3747